app/utility/transforms/OneRepMax.js

```diff
@@ -0,0 +1,10 @@
+import regression from 'regression';
+
+export const OneRMPrediction = (data, velocity) => {
+    // gather points
+    const result = regression.linear(data, { precision: 4 });
```

Comment:
why are we precision 4? what's the basis? once figured out...

- [ ] add a comment why we did precision 4